### PR TITLE
[CND] verify-libs-and-licenses

### PR DESCRIPTION
**Pull request by vieiro**
```
ant -Dcluster.config=cnd verify-libs-and-licenses
```
shows no problems now.

### CODE REVIEW

File: cnd/cnd.modelimpl/external/open-fortran-parser-0.7.1.2-notice.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Open Fortran Parser v0.7.1.2
+Copyright (c) 2005, 2006 Los Alamos National Security, LLC.
```
Comment (matthiasblaesing) [inline cnd/cnd.modelimpl/external/open-fortran-parser-0.7.1.2-notice.txt:2]: Why is this notice necessary? We include the full license, so the copyright is already present - am I missing something?

**Review by matthiasblaesing — COMMENTED** [review]
Looks sane apart from a minimal comment inline.

File: cnd/cnd.modelimpl/external/open-fortran-parser-0.7.1.2-notice.txt (hunk 1 above)
Comment (vieiro) [reply cnd/cnd.modelimpl/external/open-fortran-parser-0.7.1.2-notice.txt:2]: You're right. I'll remove/squash/push during the weekend.

**Comment by vieiro** [comment]
@matthiasblaesing , this looks nicer now without the open fortran parser notice file, doesn't it? May I merge?

**Comment by matthiasblaesing** [comment]
@vieiro sure - thank you for the update

**Pull request merged**